entity-management-web/src/main/java/eu/europeana/entitymanagement/web/JobLauncherController.java

```diff
@@ -0,0 +1,46 @@
+package eu.europeana.entitymanagement.web;
+
+import eu.europeana.entitymanagement.batch.DummyJobConfigurer;
+import eu.europeana.entitymanagement.batch.JobParameter;
+import org.springframework.batch.core.JobParameters;
+import org.springframework.batch.core.JobParametersBuilder;
+import org.springframework.batch.core.configuration.annotation.BatchConfigurer;
+import org.springframework.batch.core.launch.JobLauncher;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Date;
+
+/**
+ * Temporary controller to test Spring Batch integration.
+ * Functionality will be moved to {@link EMController}
+ */
+@RestController
+@RequestMapping(path = "/jobs")
+public class JobLauncherController {
+
+    @Autowired
+    private DummyJobConfigurer dummyJobConfigurer;
+
+    @Autowired
```

Comment:
could we use @Resource(name=)

Reply:
[The Spring team note that ](https://docs.spring.io/spring-boot/docs/2.0.x/reference/html/using-boot-spring-beans-and-dependency-injection.html)"using `@ComponentScan` (to find your beans) and using `@Autowired` (to do constructor injection) works well."

Constructor-based injection is the recommended of wiring dependencies.  `@Resource` cannot be used in this manner so it's actually the worse option.

In this case, the better approach would have been for me to use constructor injection instead.